rules/solve_myopic.smk

```diff
@@ -148,6 +148,12 @@ rule solve_sector_network_myopic:
             "sector", "co2_sequestration_potential", default=200
         ),
         custom_extra_functionality=input_custom_extra_functionality,
+        sector=config_provider("sector"),
+        limits_capacity_min=config_provider("limits_capacity_min"),
+        limits_capacity_max=config_provider("limits_capacity_max"),
+        limits_volume_min=config_provider("limits_volume_min"),
+        limits_volume_max=config_provider("limits_volume_max"),
+        co2_budget_national=config_provider("co2_budget_national"),
```

Comment:
@toniseibold just a comment: as these keys are specifications coming from the ariadne workflow (which stand-alone pypsa-eur does not use), perhaps we can rather take a modular approach and collect these entries under `solving` -> `constraints` in the ariadne repo, rather than explicitly making pypsa-eur aware of them?